### PR TITLE
Updates quota limit for car renting

### LINKED ISSUE #62: Update subquota limit classifier
There's a [new limit for renting cars](http://www2.camara.leg.br/legin/int/atomes/2017/atodamesa-183-16-maio-2017-784814-publicacaooriginal-152684-cd-mesa.html).

### PR DESCRIPTION
**Pull request by pedrovereza**
Fixes #62 

[New quota](http://www2.camara.leg.br/legin/int/atomes/2017/atodamesa-183-16-maio-2017-784814-publicacaooriginal-152684-cd-mesa.html) was defined as R$ 12.713,00.

### CODE REVIEW

**Comment by jtemporal** [comment]
Hi @pedrovereza great PR! I have a question though:

For those reimbursements from before the new definition shouldn't we have a monthly limit still somewhere in the code? I don't think the new quota value also applies to "before monthly limit increase".

@cuducos @anaschwendler what you guys think? and better yet how to properly insert this piece of information on the code?

**Comment by pedrovereza** [comment]
@jtemporal @cuducos Hey, thanks for the feedback! The change requested totally makes sense, and I think I got it right now 😉

File: rosie/chamber_of_deputies/tests/test_monthly_subquota_limit_classifier.py [hunk 1]
```diff
@@ -18,15 +18,15 @@ def setUp(self):
 
     def test_predict_false_when_not_in_date_range(self):
         assert_array_equal(np.repeat(False, 10),
-                           self.prediction[[0, 1, 9, 10, 18, 19, 27, 28, 36, 37]])
+                           self.prediction[[0, 1, 18, 19, 27, 28, 36, 37, 45, 46]])
```
Comment (pedrovereza) [inline rosie/chamber_of_deputies/tests/test_monthly_subquota_limit_classifier.py]: Quick note here: car renting before 07/2017 is not really out of range as it falls into the previous period (04/2015 - 06/2017)

**Comment by cuducos** [comment]
Yay, many thanks, @pedrovereza : ) Great contribution o/

**Pull request merged**